### PR TITLE
STN-504: Increase Traditional Font Size

### PR DESCRIPTION
**Pull request by kaseybon**
# READY FOR REVIEW

## Summary
The main goal of this PR is to update the body font in the Traditional theme to `1.8rem //18px`. Increasing the body font also required that we adjust a few other elements on the theme:

- increase heading in ptype-hs-row to 22px in the traditional theme
- set field-content font-size to 18px in traditional theme
- decrease 3rd column font size in the traditional theme
- increase hs-caption and hs-credits font size in traditional theme
- create hb-credits mixin and use for figcaption
- increase card title font-size on mobile for traditional theme

⚠️  These changes should only affect the Traditional theme
⚠️ This work is blocking STN-348

## Need Review By (Date)
asap

## Urgency
high

## Steps to Test
1. All tests should pass
1. Confirm that there are no changes to the Colorful theme
1. Confirm that the font-size of the Traditional theme is now 1.8rem (18px)
1. Confirm the following:
    - `h2`/Titles in `.hb-grid` were increased to 22px
        Home_Page___QA_Sandbox__Stage_for_Colorful__Dev_for_Traditional_
    - `.hb-card__subcontent-item` was increased to 18px
        Events___QA_Sandbox__Stage_for_Colorful__Dev_for_Traditional_
    - `.hs-caption` and `.hs-credits` were increased to 16px 
        QA_text_area___QA_Sandbox__Stage_for_Colorful__Dev_for_Traditional_
        - The `figcaption` was updated to use the `.hs-caption` styles
            QA_Image_Styles___QA_Sandbox__Stage_for_Colorful__Dev_for_Traditional_
    - The font styles in the right most column of 3 column layouts were updated to:
        ```SCSS
        font-size: 16px;
        line-height: 127%;
        ```
        Ran_Abramitzky___QA_Sandbox__Stage_for_Colorful__Dev_for_Traditional_

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)

### CODE REVIEW

File: docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.text.scss [hunk 1]
```diff
@@ -154,6 +154,14 @@
...
   line-height: 127%;
+
+  // @include hb-themes(('colorful', 'airy')) {
+  //   line-height: 127%;
+  // }
+  //
+  // @include hb-traditional {
+  //   line-height: 130%;
+  // }
 }
```
Comment (heatherdesigns) [inline docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.text.scss]: Oops! 😆   Remove commented code please.

File: docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.text.scss [hunk 2]
```diff
@@ -268,3 +276,24 @@
   }
 }
 
+@mixin hb-caption($italic: false) {
```
Comment (heatherdesigns) [inline docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.text.scss:271]: Nice addition!! :tada:   
Love the flexibility of being able to use this mixin for captions or credits but I don't see an if statement to set a `font-style` property to `italic` when the `$italic` variable is true.

On http://sparkbox-sandbox.suhumsci.loc/qa/qa-text-area Caption and Credits have the same style.

**Review by heatherdesigns — COMMENTED** [review]
Looking good! I noticed a couple things to double check.

volley

File: docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.text.scss (hunk 2 above)
Comment (kaseybon) [reply docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.text.scss:271]: 🙈 Oops!

**Review by heatherdesigns — APPROVED** [review]

**Pull request merged**